sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/signaling/SignalingConfig.java

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.chat.signaling;
+
+class SignalingConfig {
+
+    public static final String TROUTER_CLIENT_VERSION = "99116/0.0.0.0/";
+    public static final String DEFAULT_TROUTER_HOSTNAME = "go.trouter-int.skype.net";
+    public static final String DEFAULT_REGISTRATION_HOSTNAME_AND_BASE_PATH = "edge.skype.net/registrar/testenv";
```

Comment:
Consider load these values from a config file. These values are not there for production use, are they?

Reply:
I think they are for production use. They are probably used to configure the real time channel.

Reply:
If that's the case, we should leave it as is. Hard coded stuff makes me nervous. :-)